config/jobs/periodic/docker/periodic-ci-docker.yaml

```diff
@@ -0,0 +1,207 @@
+periodics:
```

Comment:
I hope this PR restores the CI jobs that were part of [#447](https://github.com/ppc64le-cloud/test-infra/pull/447).

Reply:
Yes